tests/test_api.py

```diff
@@ -301,7 +301,7 @@ async def test_sslcontext_api_success_async(host):
     ctx = truststore.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
     # connector avoids https://github.com/aio-libs/aiohttp/issues/5426
     async with aiohttp.ClientSession(
-        connector=aiohttp.TCPConnector(force_close=True, enable_cleanup_closed=True)
+        connector=aiohttp.TCPConnector(force_close=True)
```

Comment:
FYI: `enable_cleanup_closed=True` is still required on Python <=3.11.

#168 filtered the deprecation warning causing this to fail.

Reply:
I rebased on latest main and dropped my change for this, thanks a lot :)